### ISSUE TITLE
provide download-links for content tables in <div id="notes">

### ISSUE THREAD

**Issue by cosmowiki** [cosmowiki/cosmowiki#17]
@wolframkriesing 
maybe as .csv instead of .json so users can import and edit the tables

**Comment by wolframkriesing** [comment]
i like the idea.
i think we should offer the data in a proper structure, not the structure that the current JSON is in, right?
The names like `chronicley` etc. just are not too self-speaking, right? What do you think if this becomes simply another view that just exposes the names of the well-named objects as they are constructed now in the code, like this one https://github.com/cosmowiki/cosmowiki/blob/36732cace81927babd09bdbdef9a6804c6fde747/src/sites/events.js#L16-L39 ? which then just gets converted into a well-readable JSON and CSV?

**Comment by cosmowiki** [comment]
Right, that’s why i prefered a .csv. The shortened names are not easy to understand, but I can change them. We should offer a format which can be edited by users and sent back (or better edited online?) so we can use it directly w/o a big conversion for CW. So the question is if the declarations for year, month and day should be separated as they are now or we find a way to use merged dates in cases like BC or so. You know the difficulties especially in Chronicle ;-)

Mit freundlichen Grüßen / Kind regards
Holger Porsch
CosmoWiki.de

--
web		http://CosmoWiki.de/ <http://cosmowiki.de/>

**Comment by wolframkriesing** [comment]
The current input can stay as it is, I would suggest we generate the data
from what we have and then offer an editor or alike. But first we can
provide the data nicely formatted from what we have.
A crucial thing is, I would say, to provide a stable interface to the
outside world to first allow reading the data, and later allow
contributions.
Am 07.10.2015 14:52 schrieb "cosmowiki" <notifications@github.com>:

**Comment by cosmowiki** [comment]
Why not letting the users open the file in google spreadsheets from where they're able to send it back via mail or store it right there. I don't know what's possible.

**Comment by wolframkriesing** [comment]
That is a really cool idea
Am 07.10.2015 16:49 schrieb "cosmowiki" <notifications@github.com>:
